### ISSUE TITLE
onClick is called on drag start

### ISSUE THREAD

**Issue by Lewik** [mipt-npm/maps-kt#8]
onClick is called on pointer down
It leads to onClick execution at drag start

**Comment by altavir** [comment]
Yes, I am not sure how to differentiate between them. We can call onClick when the pointer mouse button released instead on down. Will it make sense?

**Comment by Lewik** [comment]
Yes, it should call onClick on release.
Also i found this builtin way:
https://github.com/Lewik/mapview/blob/148dc3508d0b133189d1a68df57a7df056997c84/common/src/desktopMain/kotlin/mapview/canvasGestures.kt

**Comment by altavir** [comment]
Thanks, I will try it later
